File: Data Structures & Algorithms/longest-repeating-substring-with-replacement/submission-14.cpp
class Solution {
public:
    int characterReplacement(std::string s, int k) {
        unordered_map<char, int> m;;
        int res = 0, mf = 0, l = 0, r = 0;
        for (int r = 0; r < s.size(); r++) {
            m[s[r]]++;
            mf = max(mf, m[s[r]]);
            while (r-l+1-k >mf) {
                m[s[l]]--;
                l++;
            }
            res = max(res, r-l+1);
        }
        return res;
    }
};

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-14.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        n = set(nums)
        c = 0 
        for i in range(len(nums)):
            count = 1
            nu = nums[i]
            if nu+1 not in nums:
                while nu-1 in n:
                    count+=1
                    nu-=1
            c = max(count, c)

        return c